WeatherApp/WeatherApp/Core/SceneDelegate.swift

```diff
@@ -3,7 +3,7 @@ import SwiftUI
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
     var window: UIWindow?
-    var router: Router!
+    lazy var dependencies = Dependencies()
```

Comment:
I believe you could private both vars

Reply:
Dependencies can be private, but window can't be. It then requests to be internal.